src/Symfony/Component/VarExporter/Internal/LazyObjectState.php

```diff
@@ -61,6 +62,10 @@ public function initialize($instance, $propertyName, $propertyScope)
             $this->status = self::STATUS_UNINITIALIZED_FULL;
             $this->reset($instance);
 
+            if ($e instanceof \Error && 1 === preg_match('#^Cannot modify readonly property (.+)$#', $e->getMessage(), $matches)) {
+                throw new LogicException(sprintf('Lazy ghost cannot reinitialize readonly property "%s", remove the readonly modifier on that property and/or its containing class, or ensure the ghost will never be reinitialized.', $matches[1]));
```

Comment:
```suggestion
            if ($e::class === \Error::class && preg_match('#^Cannot modify readonly property (.+)$#', $e->getMessage(), $m)) {
                throw new LogicException(sprintf('Lazy ghost cannot reinitialize property "%s" because it is readonly.', $m[1]));
```